python/ray/serve/tests/test_cross_language.py

```diff
@@ -4,7 +4,7 @@
 from ray.serve.config import ReplicaConfig, DeploymentConfig
 from ray.serve.utils import msgpack_serialize
 from ray.serve.generated.serve_pb2 import JAVA, RequestMetadata, RequestWrapper
-from ray.tests.conftest import shutdown_only  # noqa: F401
+from ray.tests.conftest import shutdown_only, maybe_external_redis  # noqa: F401
```

Comment:
Is `maybe_external_redis` used?

Reply:
I'm not sure what's happening here, but if I didn't include `maybe_external_redis` it will complains about can't find it.
I think the previous F401 tag maybe is because of this.